packages/aws-rfdk/lib/lambdas/nodejs/ecr-provider/handler.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-console */
+
+import * as path from 'path';
+
+import { LambdaContext } from '../lib/aws-lambda';
+import { CfnRequestEvent, SimpleCustomResource } from '../lib/custom-resource';
+import {
+  ThinkboxEcrProvider,
+} from '../lib/ecr-provider';
+
+/**
+ * The input to this Custom Resource
+ */
+export interface ThinkboxEcrProviderResourceProperties {
+  /**
+   * A random string that forces the Lambda to run again and obtain the latest ECR.
+   */
+  readonly ForceRun?: string;
+}
+
+/**
+ * Output of this Custom Resource.
+ */
+export interface ThinkboxEcrProviderResourceOutput {
+  /**
+   * The ARN prefix of the ECR repository. This can be suffixed with the recipe name to get the final ECR repository
+   * ARN.
+   */
+  readonly EcrArnPrefix: string;
+}
+
+/**
+ * This custom resource will parse and return the base ECR ARN or URI containing Thinkbox published Docker Images.
+ * A global ECR base URI is returned.
+ */
+export class ThinkboxEcrProviderResource extends SimpleCustomResource {
+  readonly ecrProvider: ThinkboxEcrProvider;
+
+  constructor() {
+    super();
+    this.ecrProvider = new ThinkboxEcrProvider(path.join(
+      __dirname,
+      '..',
+      'lib',
+      'ecr-provider',
+      'mock-data.json',
```

Comment:
Just confirming that this is a correct filename, not related to testing etc.

Reply:
Yes, this is a placeholder for now. It will be removed once we have the official index file published. Once that change is made, this PR will be removed from draft and I'll notify here.